FRC1710-2019/src/main/java/frc/robot/intakePOS.java

```diff
@@ -0,0 +1,91 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+package frc.robot;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+import edu.wpi.first.wpilibj.Joystick;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class intakePOS extends Command {
+
+  public static TalonSRX IntakeArmAxsisR;
+  public static TalonSRX IntakeArmAxsisL;
+  
+  private static final int LEFT_ID = 5; //Replace with current id
+  private static final int RIGHT_ID = 6; //Replace with current id
+
+  private double AxisPOS = 0.0; // the axis position of just one of the motors
+  
+  private static final double ENC_resting_POS = 0.0;  //where the arm actually is
+  private static final double ENC_ballIntake_POS = 0.0;
+  private static final double ENC_hatchIntake_POS = 0.0;
```

Comment:
These should characters should all be upper case: ENC_RESTING_POS

See https://www.oracle.com/technetwork/java/codeconventions-135099.html

Reply:
Similarly, can we use the full word POSITION? No reason to abbreviate. We should strive to make the code as easy to read and understand as possible.